e2e/pages/CreatePage.js

```diff
@@ -26,20 +25,23 @@ module.exports = (url) => (driver) => ({
             .then(() => driver.sleep(100)); // let some time to redraw
     },
 
-    setTitleValue(value, clearPreviousValue = true) {
-        const input = driver.findElement(this.elements.titleInput);
+    setInputValue(type, name, value, clearPreviousValue = true) {
+        const input = driver.findElement(this.elements.input(type, name));
         if (clearPreviousValue) {
             input.clear();
         }
         return input.sendKeys(value);
     },
-    
-    setTeaserValue(value, clearPreviousValue = true) {
-        const input = driver.findElement(this.elements.teaserInput);
-        if (clearPreviousValue) {
-            input.clear();
-        }
-        return input.sendKeys(value);
+
+    setValues(values, descValue="", clearPreviousValue = true){
```

Comment:
why `descValue`? Also, be consistent with spacing. In default values, it's `descValue = "",`